02-loading-data/from_internet.Rmd

```diff
@@ -10,15 +10,17 @@ output: html_document
 
 If you find data that you would like to use on a website, package `rvest` can bring that table into R. (Scraping data both saves you the process/frustration of copying/pasting data from a webpage, and also makes your work less error-prone and more reproducible.)
 
-NOTE: Before you take data from a website, make sure you are allowed to scrape and analyze the data.
+#### Checking for Permission
 
-Luckily there is a package that makes this data scraping process fairly straightforward. First, install the `robotstxt` package:
+Before you take data from a website, always make sure you are allowed to scrape and analyze the data. You can sometimes find this information by digging into the webpage, but luckily there's an R-package that will do this for you. It is called `robotstxt` and while it has a great number of functions, the one that we'll be using is called `paths_allowed()`.
```

Comment:
this is a great explanation!

Reply:
seconding Ingrid! This looks great :) I was looking at Ava's comment about visuals for the scraping section and I agree! Especially with the "inspect" thing, when I tried this on my Mac, that option wasn't there and I had to go into my setting and change it. Maybe we can add some visuals on where you should see that and a disclaimer of "if you don't see this, follow these steps". I use safari on Mac and I have the screenshots on how to make sure you have the inspect option but it might be different for others browsers/computers. I added those to a folder in the shared drive -- mostly because I am not super sure how to add images and such to R/Git